build_scripts/appimage/install-infection-monkey-service.sh

```diff
@@ -31,13 +32,20 @@ echo_help() {
 }
 
 install_service() {
+  exit_if_service_installed
   copy_appimage
   install_systemd_unit "$1"
 
   echo "The Infection Monkey service has been installed and will start on boot."
   echo "Run 'systemctl start infection-monkey' to start the service now."
 }
 
+exit_if_service_installed() {
+  if sudo systemctl list-units --full --all | grep -Fq "${SYSTEMD_UNIT_FILENAME}"; then
+    die "Error: Service ${SYSTEMD_UNIT_FILENAME} is already installed."
```

Comment:
Perhaps we should add uninstallation instructions? "Please uninstall with `sudo ... service --uninstall` before attempting to install" or something like that?